Application/Handlers/GetTuitionPartnerQueryHandler.cs

```diff
@@ -29,7 +40,7 @@ public class GetTuitionPartnerQueryHandler : IRequestHandler<GetTuitionPartnerQu
         var tp = tpResult.Result.Data.FirstResult;
 
         var subjects = tp.SubjectsCoverage!.Select(x => x.Subject).Distinct().GroupBy(x => x.KeyStageId)
-            .Select(x => $"{((Enums.KeyStage)x.Key).DisplayName()} - {x.DisplayList()}");
+            .Select(x => $"{EnumExtensions.DisplayName(((Enums.KeyStage)x.Key))} - {x.DisplayList()}");
```

Comment:
Is there a reason behind not using the `DisplayName` as an extension method here?

Reply:
We should be able to use the `DisplayName` as an extension method just fine. I will correct that now. Thanks!